css/style.css

```diff
@@ -51,41 +40,84 @@ p {
 
 }
 
-.header-logo { 
-    margin: 0 auto;
+.header__logo { 
+    display: flex;
+    align-items: center;
 }
 
-.header-text {
+.header__links-group {
     display: flex;  
     gap: 70px;
+    align-items: center;
+    height: 100%;
 }
 
-.header-text-link {
+.header__link {
     color: #342B2B;
     font-size: 16px;
 }
 
+.header__link-with-image {
+    display: flex;
+    gap: 9px;
+}
+
 
 /* Button */
-.button {
-    margin: 0 auto;
+.header__buttons-group {
+    font-family: Rubik, sans-serif;
+    font-size: 16px;
+
+    display: flex;
+    gap: 6px;
+    align-items: center;
+    height: 100%;
+    justify-content: flex-end;
+
 }
 
-.button1 {
+.header__button-register {
+    font-family: Rubik, sans-serif;
     padding: 6px 24px;
 
     background-color: #FFFFFF;
-
     color: #1E3DAF;
     font-size: 16px;
+
+    border: 2px solid #1E3DAF;
+    border-radius: 4px;
 }
 
-.button2 {
+.header__button-register:hover {
+    cursor: pointer;
+}
+
+.header__button-login {
+    font-family: Rubik, sans-serif;
     padding: 6px 24px;
-    margin-right: 375px;
 
     background-color: #1E3DAF;
-
     color: #FFFFFF;
     font-size: 16px;
-}
\ No newline at end of file
+
+    border: 2px solid #1E3DAF;
+    border-radius: 4px;
+}
+
+.header__button-login:hover {
```

Comment:
don't need to add it on hover. You can just add it in the `header__button-login` directly